alphabet/az/az_contract.go

```diff
@@ -111,6 +111,24 @@ func Emit() bool {
 	return true
 }
 
+func Vote(candidate []byte) {
+	innerRingKeys := irList()
+	if !checkPermission(innerRingKeys) {
```

Comment:
This means that misbehaving IR node can vote for some random consensus node. I've actually thought that this method is to be invoked from IR multisignature address. And that there is 1:1 mapping between CN nodes and alphabet contracts rather than 1:1 IR/alphabet.

Reply:
After discussion we've decided to:
- invoke `Vote` method with **the list of candidates**,
- accumulate signatures from **first seven** inner ring nodes in contract,
- vote for one candidate from the list as soon as signature threshold is reached.